app/src/main/java/uc/edu/ozturkkl/traveltracker/services/LocationService.kt

```diff
@@ -24,7 +24,9 @@ class LocationService {
                 call: Call<ArrayList<PredictionResponse>>,
                 response: Response<ArrayList<PredictionResponse>>
             ) {
-                _locations.value = response.body()
+                if (Response.isSuccessful()) {
```

Comment:
I don't think this will work, as the capitalization here is different than the capitalization of the parameter variable.  Thus, this won't compile.